app/addons/dashboard/components.react.jsx

```diff
@@ -0,0 +1,190 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'react',
+  'addons/dashboard/stores'
+], function (App, FauxtonAPI, React, Store) {
+
+  var activeTaskList = Store.dashboardStore;
+
+  var DashboardController = React.createClass({
+
+    getStoreState: function () {
+      return {
+        collection: activeTaskList.getCollection(),
+
+        setPolling: activeTaskList.setPolling,
+        clearPolling: activeTaskList.clearPolling
+      };
+    },
+
+    getInitialState: function () {
+      return this.getStoreState();
+    },
+
+    componentDidMount: function () {
+      this.state.setPolling();
+      activeTaskList.on('change', this.onChange, this);
+    },
+
+    componentWillUnmount: function () {
+      this.state.clearPolling();
+      activeTaskList.off('change', this.onChange, this);
+    },
+
+    onChange: function () {
+      this.setState(this.getStoreState());
+    },
+
+    render: function () {
+      var collection = this.state.collection;
+      var isEmpty = _.isEmpty(collection);
+      return (
+        <div>
+          <ActiveTaskWidget collection={collection} isEmpty={isEmpty}/>
+        </div>
+        );
+    }
+  });
+
+  var ActiveTaskWidget = React.createClass({
+
+    render: function () {
+      var collection = this.props.collection;
+      var isEmpty = this.props.isEmpty;
+      return (
+        <div className="widget-container active-task-box">
+          <div className="widget-header">
+            <span>Active Tasks</span>
+          </div>
+          <hr className="widget-header-separator"/>
+          <div className="active-task-table">
+            <ActiveTaskTable collection={collection} isEmpty={isEmpty}/>
+          </div>
+        </div>
+        );
+    }
+  });
+
+  var ActiveTaskTable = React.createClass({
+
+    getStoreState: function () {
+      return {
+        filteredTable: activeTaskList.getFilteredTable(this.props.collection)
+      };
+    },
+
+    getInitialState: function () {
+      return this.getStoreState();
+    },
+
+    componentWillReceiveProps: function (nextProps) {
+      this.setState({
+        filteredTable: activeTaskList.getFilteredTable(this.props.collection)
+      });
+    },
+
+    createRows: function () {
+      var isThereResults = this.props.isEmpty;
```

Comment:
I think this var name is inverted. I'd probably just remove it altogether and use `this.props.isEmpty` directly in the code that follows.